examples/tests/sdk_tests/website/website.w

```diff
@@ -0,0 +1,24 @@
+bring cloud;
+
+let w = new cloud.Website(path: "./website");
+let config = { json: 1 };
+
+class Util {
+    extern "../external/http.js" static inflight http(url: str): Json;
+    extern "../external/fs.js" static readFile(path: str): str;    
+  }
+ 
+let indexFile = Util.readFile("./website/website/index.html");
+let otherFile = Util.readFile("./website/website/inner-folder/other.html");
+
+w.addJson("config.json", config);
+
+assert(w.path.endsWith("sdk_tests/website/website"));
+
+
+test "test" {
+    // doesn't work on tf-aws, we need to create an inflight method for retrieveing the url (there are no inflight properties)
```

Comment:
I'm not sure I understand the comment - the `.url` field is defined in preflight but it should still be accessible in inflight, so we shouldn't need to add any new inflight methods or properties. Is the issue that the token is not getting resolved (similar to https://github.com/winglang/wing/issues/2232)?

Reply:
it's resolves well in sim and as undefined in aws... opened an issue #2829